Parameter.java

```diff
@@ -0,0 +1,10 @@
+package parameter;
+
+public class Parameter {
+	 public static void main(String args[])
```

Comment:
Try to pass parameters in by using array 'args[]'. 

Follow example on http://docs.oracle.com/javase/tutorial/essential/environment/cmdLineArgs.html